docs/usage/configuration/config_documentation.md

```diff
@@ -1212,6 +1212,14 @@ like sending a federation transaction.
 * `max_short_retries`: maximum number of retries for the short retry algo. Default to 3 attempts.
 * `max_long_retries`: maximum number of retries for the long retry algo. Default to 10 attempts.
 
+The following options are related to configuring the backoff parameters used for a specific destination.
+Unlike previous configuration options, these values apply across all requests
+and the state of the backoff is stored in the database.
```

Comment:
Probably prefer "retry logic" over "backoff" since we use it above and the options have `retry` in name. I think "backoff" makes sense in the second sentence.

```suggestion
The following options control the retry logic when communicating with a specific homeserver destination.
Unlike the previous configuration options, these values apply across all requests
for a given destination and the state of the backoff is stored in the database.
```